views/table.ejs

```diff
@@ -0,0 +1,151 @@
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
+<br>
+<a href="/event/view/<%=event_id%>"><button class="btn btn-primary">Return to event</button> </a>
+<body class="container">		
+	<br>
+	<div style="text-align: center;font-size:2em;"><%=eventName%> - Table Plan</div>
+	<br>
+	<% if (succ && succ.length > 0) { %>
+		<div class="alert alert-success" role="alert">
+			<%= succ %>
+		</div>
+	<% } %>
+	<% if(table.length>0){%>
+	<div class= "row">			
+			<% for(var i=0; i < table.length; i++) { %>
+			<div class="col-md-4" style="padding:5px;">
+				<div class="card" style="width:20rem; padding:15px;border-color:#0181C6; border-style:solid; border-radius:8px; border-width:medium">
+				<div class="card-body">
+					<h5 class="card-title" style="text-align: center;">Table <%= table[i].tableNumber%></h5>
+					<h6 class="card-subtitle mb-2 text-muted" style="text-align: center;">
+							<% for(var j=0; j < table[i].tableLabels.length; j++) { %>
+								<%= table[i].tableLabels[j]%>
+								<%if(j!=table[i].tableLabels.length-1){%>
+								,
+								<%}%>
+
+							<% }%> 
+					</h6>
+					<p class="card-text">
+							<% for(var x=0; x < table[i].seatLabels.length; x++) { %>
+								Seat <%= x+1%> : <%= table[i].seatLabels[x]%><br>
+							<% }%> 
+					</p>		
+					<button type="button" class="btn btn-warning" data-toggle="modal" data-target="#myModal<%=i%>">Edit</button>				
+					<button type="button" class="btn btn-danger" data-toggle="modal" data-target="#deleteModal<%=i%>">Delete</button>							
+					<!-- Edit Model-->
+					<form action="/table/edit/<%= table[0].eventId %>/<%= table[i].tableNumber %>/<%=table[0].tableLabels.length %>" method="post" id='form'>
+								<div id="myModal<%=i%>" class="modal fade" role="dialog">
+									<div class="modal-dialog">
+										<div class="modal-content">
+											<div class="modal-header">
+												<button type="button"style="float:left;margin:0;" class="close" data-dismiss="modal">&times;</button>
+												<h4 class="modal-title" style="float:left">Table <%= table[i].tableNumber%></h4>
+											</div>
+											<div style="margin:10px; text-align: center;">Table Labels</div>
+											<div id="labelGroup<%=i%>" style="padding:2px">
+											<% for(var j=0; j < table[i].tableLabels.length; j++) { %>
+												
+													<input type="name" name="label-<%=j%>" class="labels<%=j%> form-control" value="<%= table[i].tableLabels[j]%>" name="label" rows="5" required><br>												
+												
+											<%}%> 
+											</div>	
+											<% if(table[i].tableLabels.length ==0){%>
+												<div id="noLabels<%=i%>" style="margin:10px; text-align: center;">-No labels-</div>
+											<%}%>
+											<div data-table="<%=i%>" class="addLabel btn btn-secondary btn-sm">Add label</div>
```

Comment:
As mentioned in message: add label functionality is not as I would expect and cannot be recovered from